src/components/Molecules/VideoBanner/VideoBanner.md

```diff
@@ -0,0 +1,7 @@
+```js
+const poster = require('../../../styleguide/assets/poster.png');
+const src =
+  'https://www.comicrelief.com/sites/default/files/downloads/Creativists_Logo_Web_small_V2_0.mp4';
```

Comment:
Are we able to add this to the repo too?

Reply:
it's 2.4mb .. let's not save it locally at this point